src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -119,12 +119,13 @@ bipyramid(P::Polyhedron{T}, z::Number, z_prime::FieldElem) where {T<:scalar_type
   bipyramid(P, parent(z_prime)(z), z_prime)
 
 @doc raw"""
-    normal_cone(P::Polyhedron, i::Int64)
+    normal_cone(P::Polyhedron{T}, F::AbstractVector; outer::Bool = false)
```

Comment:
Mentioning `{T}` seems redundant
```suggestion
    normal_cone(P::Polyhedron, F::AbstractVector; outer::Bool = false)
```
Should the old signature perhaps still be documented?
```suggestion
    normal_cone(P::Polyhedron, i::Int64)
    normal_cone(P::Polyhedron, F::AbstractVector; outer::Bool = false)
```

Reply:
It was like that before and I am not sure what T actually changes, (would be great to have a little explanation).

But I also think it would be better to include the old signature in the documentation, since input doesn't necessarily has to be a vector.